ntbs-service/wwwroot/source/Components/YearComparison.ts

```diff
@@ -0,0 +1,38 @@
+import Vue from 'vue';

```

Comment:
This is for the BCG Year to birth year comparison... unfortunately requires its own vue component as we are comparing unrelated data,

Reply:
yeah, that makes perfect sense.